src/main/kotlin/net/maizegenetics/phgv2/cli/CreateFastaFromHvcf.kt

```diff
@@ -40,11 +43,11 @@ class CreateFastaFromHvcf : CliktCommand( help = "Create a FASTA file from a h.v
     val dbPath by option(help = "Folder name where TileDB datasets and AGC record is stored.  If not provided, the current working directory is used")
         .default("")
 
-    val output by option("-o", "--output", help = "Name for output Fasta file")
+    val outputDir by option("-o", "--output-dir", help = "DIrectory where output fasta FIles will be written")
```

Comment:
Typos in Directory and in Files

Reply:
Also does this need both names?  or will --output-dir be automatically picked up?  A unit test could check this quickly.

Reply:
This command was originally written with both options, which yes, I think should be mutually exclusive.  I ignored that while fixing the output bug, but could address it as part of this pull request.

Reply:
I misread your comment .  You were asking if both -o and --output-dir were needed.  Don't know, will test.  My change was to match outputDir with output-dir, which replaced the previous "output" parameter.